pages/apply.html

```diff
@@ -0,0 +1,364 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html><head>
+<link rel="shortcut icon" href="https://ssl.gstatic.com/docs/spreadsheets/forms/favicon_qp2.png" type="image/x-icon">
+<meta http-equiv="Content-type" content="text/html; charset=utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=10; chrome=1;">
+<meta name="fragment" content="!">
+<meta name="viewport" content="width=device-width">
+<base target="_blank">
+<title>Apply to Join 18F</title>
+<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:400,700">
+
+<link rel="stylesheet" type="text/css" href="../assets/css/all.css">
+<link rel="stylesheet" type="text/css" href="../assets/css/form.css">
+
+<script type="text/javascript" src="../assets/js/jquery-1.11.3.min.js"></script>
```

Comment:
The Standards are [now using version 2.2.0](https://github.com/18F/web-design-standards/blob/b953590cbd9712bd22b6477893b9de10957ba574/package.json). May update this accordingly, reviewing the markup for continued compatibility.